_posts/2016-05-20-drupalcon-nola-recap.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Drupalcon New Orleans 2016: a Twitter recap"
+date: 2016-05-20
+author: Chris Russo
+tags: drupal drupal8 drupalcon event
+summary: The ins and outs of the talks, sessions, parties, and more from Drupalcon New Orleans 2016.
```

Comment:
@chrisarusso - You'll need to create the tag for `drupalcon`. I forget what files you need to create/update but just search for "drupal8". Otherwise, the `drupalcon` tag appears as "Druapl 8" and links to Drupal 8 posts as well. You can see this on staging: http://blabs.savasdev.com/2016/05/20/drupalcon-nola-recap.html